app/src/main/java/com/sopt/now/MainActivity.kt

```diff
@@ -1,15 +1,31 @@
 package com.sopt.now
 
 import android.os.Bundle
-import androidx.activity.enableEdgeToEdge
 import androidx.appcompat.app.AppCompatActivity
-import androidx.core.view.ViewCompat
-import androidx.core.view.WindowInsetsCompat
+import com.sopt.now.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityMainBinding
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        setUserInfo()
+    }
+
+    // 사용자 정보 세팅
+    private fun setUserInfo(){
+        val userId = intent.getStringExtra("userId")
+        val userPw = intent.getStringExtra("userPw")
+        val userName = intent.getStringExtra("userName")
+        val userMbti = intent.getStringExtra("userMbti")
 
+        binding.userId.text = userId
+        binding.userPw.text = userPw
+        binding.tvName.text = userName
+        binding.tvMbti.text = userMbti
```

Comment:
```suggestion
        with(binding){
        userId.text = userId
        userPw.text = userPw
        tvName.text = userName
        tvMbti.text = userMbti
        }
```
with라는 scope함수를 통해 좀더 간편하게 코드를 작성 할 수 있을 것 같아요!